lib/actions/actions.js

```diff
@@ -35,17 +34,18 @@ actions._prepCopy = function _prepCopy(source, destination, process) {
   destination = this.isPathAbsolute(destination) ? destination : path.join(this.destinationRoot(), destination);
 
   var encoding = null;
-  var binary = isBinaryFile(source);
-  if (!binary) {
-    encoding = 'utf8';
-  }
+  var body = fs.readFileSync(source);
+
+  var isText = istextorbinary.isTextSync(undefined, body);
 
-  body = fs.readFileSync(source, encoding);
+  if (isText) {
+    body = body.toString();
 
-  if (typeof process === 'function' && !binary) {
-    body = process(body, source, destination, {
-      encoding: encoding
-    });
+    if (typeof process === 'function') {
```

Comment:
That's mostly legacy feature. You'll have the capacity to edit binary content using the new file system (package is mem-fs-editor).